backend/src/main/java/com/javahelp/backend/endpoint/search/SearchHandler.java

```diff
@@ -26,33 +31,35 @@
 import jakarta.json.JsonObject;
 import jakarta.json.JsonObjectBuilder;
 
-public class SearchHandler extends HTTPHandler implements ISearchInput {
+/**
+ * Handler for searches for providers
+ */
+public class SearchHandler extends HTTPTokenHandler implements ISearchInputBoundary {
```

Comment:
Amazing to see SearchHandler now extending HTTTokenHandler instead of just HTTPHandler.